prisma/schema.prisma

```diff
@@ -78,23 +76,23 @@ model User {
   id            String     @id @default(cuid())
   name          String?
   email         String?    @unique
-  emailVerified DateTime?
   image         String?
-  role          UserRole        @default(PROGRAM_ADMINISTRATOR)
+  emailVerified DateTime?
+  role          UserRole   @default(PROGRAM_ADMINISTRATOR)
   accounts      Account[]
-  adminLogs    AdminLog[]
+  adminLogs     AdminLog[]
   sessions      Session[]
 }
 
-enum UserRole {
-  ADMINISTRATOR
-  PROGRAM_ADMINISTRATOR
-}
-
 model VerificationToken {
   identifier String
   token      String   @unique
   expires    DateTime
 
   @@unique([identifier, token])
 }
+
+enum UserRole {
```

Comment:
Any particular reason this got moved down a few lines?

Reply:
It's the autogenerated schema file using the new prisma version.  Looks like Prisma prefers to have them at the very end of the file as a best practice.